airflow-core/src/airflow/ui/public/i18n/locales/nl/components.json

```diff
@@ -1,6 +1,6 @@
 {
   "backfill": {
-    "affected_one": "1 run zal getriggered worden.",
+    "affected_one": "1 run zal getriggerd worden.",
```

Comment:
I wonder whether this is a good suggestion (spelling?) But it is not changed below 🤔